src/MSBuildTaskHost/MSBuildTaskHost.csproj

```diff
@@ -204,7 +204,6 @@
     <ProjectReference Include="..\StringTools\StringTools.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" />
```

Comment:
I think this will be good, but please do an exp branch, install the val build, and build some 3.5 projects to exercise taskhost scenarios.

Reply:
On it https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=4784041&view=results

Reply:
The build finished. Did you ever play around with it?

Reply:
Just installed the net3.5 dev packs on the VM. Will post when its tested